docs/source/install/index.rst

```diff
@@ -53,7 +53,7 @@ on Ubuntu, or ``sudo yum install curl nss`` on RHEL/CentOS. Then run this comman
 
 .. code-block:: bash
 
-   curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password=Ch@ngeMe
+   bash <(curl -sSL https://stackstorm.com/packages/install.sh) --user=st2admin --password=Ch@ngeMe --unstable --u16-add-insecure-py3-ppa
```

Comment:
Are we sure we want to include `--unstable` here? Or is tweaking the `bash <(curl ...)` and adding the `--u16-add-insecure-py3-ppa` flag enough?

I'd like this change to be as small as humanly possible, since we are soooooo close to release.

Reply:
I actually removed the `--add-insecure-py3-ppa` and `--unstable` flag with the most recent commit. So we don't change anything on the behaviour or results but support interactive scripts now. I'm sorry for the confusion.